### PR TITLE
docs: Fix spelling and formatting

### PR DESCRIPTION
**Pull request by jodh-intel**
Correct typos and resolve formatting issues including incorrect heading levels and missing TOC entries.

Fixes: #541.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

File: kata-deploy/README.md [hunk 1]
```diff
@@ -24,83 +23,83 @@ a node only if it uses either containerd or CRI-O CRI-shims.
...
+- `/var/run/dbus`, `/run/systemd`: this is require for reloading the the Docker service
+- `/etc/docker`: this is required for updating `daemon.json` in order to configure the Kata runtimes in Docker
 
 
 ### Install Kata and configure Docker
 
 To install:
 
 ```
-docker run -v /opt/kata:/opt/kata -v /var/run/dbus:/var/run/dbus -v /run/systemd:/run/systemd -v /etc/docker:/etc/docker -it katadocker/kata-deploy kata-deploy-docker install
```
Comment (grahamwhaley) [inline kata-deploy/README.md]: could `sh` ize the block type..

File: snap/README.md [hunk 2]
```diff
@@ -60,11 +60,11 @@ $ sudo cp /snap/kata-containers/current/usr/share/defaults/kata-containers/confi
...
 
-## Integration with docker and kubernetes
+## Integration with docker and Kubernetes
 
 The path to the runtime provided by the Kata Containers snap image is
 `/snap/kata-containers/current/usr/bin/kata-runtime`. You should use it to
-run Kata Containers with [docker][9] and [kubernetes][10].
+run Kata Containers with [docker][9] and [Kubernetes][10].
 
 ## Remove snap
```
Comment (grahamwhaley) [inline snap/README.md]: You capitalised the toc, do you need to match the section header (and maybe any of the body) here?
Whilst here, afaict Ubuntu never capitalise 'snap' ?

File: kata-deploy/README.md (hunk 1 above)
Comment (jodh-intel) [reply kata-deploy/README.md]: Agreed - fixed (also yaml blocks).

File: snap/README.md (hunk 2 above)
Comment (jodh-intel) [reply snap/README.md]: I don't follow? This section heading does match the toc (which I actually lower-cased)...?

Comment (grahamwhaley) [reply snap/README.md]: ay, yep, I read the patch backwards ;-) np

**Comment by jodh-intel** [comment]
Gah - this gets me every time! :smile:

```
Kernel directory has changes check kernel/kata_config_version changed
ERROR: Please bump version in kernel/kata_config_version
```

/retest

**Pull request merged**